controllers/jwtDecode.js

```diff
@@ -0,0 +1,8 @@
+const jwt = require('jsonwebtoken');
+
+const jwtDecode = async (req, res, next) => {
```

Comment:
What if the token is expired or someone has changed the token. You are still allowing the user to reach the api level.
Try to block the user if that is the case. Throw an error. If the token is expired or malformed.